README.md

```diff
@@ -83,9 +83,28 @@ Unsubscribing people:
 Listing people:
 
 ```php
-// List all people, 20 per page, first 2 pages
-$people = \Delighted\Person::all()
-$people_p2 = \Delighted\Person::all(['page' => 2]);
+// List all people, auto pagination
+// Note: Make sure to handle the possible rate limits error
+$people = \Delighted\Person::list();
+while (true) {
+  try {
+    foreach ($people->autoPagingIterator() as $person) {
+      // Do something with $person
+    }
+    break;
+  } catch (\Delighted\RequestException $e) {
+    // Indicates how long (in seconds) to wait before making this request again
+    $e->getRetryAfter();
+    continue;
+  }
+}
+
+// For convenience, this method can use a sleep to automatically handle rate limits
+$people = \Delighted\Person::list(['auto_handle_rate_limits' => true]);
```

Comment:
Not really a [standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md#42-properties) for variable names, community is a bit free for all there, I'm used to underscores and was gonna say consistency wins over, but we don't really have consistency there either! I'd keep it as `_` since we have a bit more snake_case, but open to ideas.